DSCResources/xTimeZone/xTimeZone.psm1

```diff
@@ -13,7 +13,7 @@ function Get-TargetResource
     )
 
     #Get the current TimeZone
-    $CurrentTimeZone = Invoke-Expression "tzutil.exe /g"

```

Comment:
Perhaps, Windows line ending vs Linux line ending (CRLF vs LF)? You can check both versions with some hex editor.